upup/pkg/fi/cloudup/awstasks/load_balancer.go

```diff
@@ -362,10 +477,11 @@ func (_ *LoadBalancer) RenderTerraform(t *terraform.TerraformTarget, a, e, chang
 		elbName = e.Name
 	}
 
-	internal := fi.StringValue(e.Scheme) == "internal"
 	tf := &terraformLoadBalancer{
-		Name:     elbName,
-		Internal: internal,
+		Name: elbName,
+	}
+	if fi.StringValue(e.Scheme) == "internal" {
```

Comment:
FYI that's not a cast, that's a helper function...

This PR is obsolete anyway ... I forgot we'd split out pieces of the mega PR